SIL.Media/Naudio/AudioRecorder.cs

```diff
@@ -215,6 +215,19 @@ public virtual void BeginMonitoring()
 
 		}
```

Comment:
Would it be better to have BeginMonitoring call BeginMonitoringWithThrow so as not to duplicate the little bit of logic and exception throwing? i.e.
`public virtual void BeginMonitoring()
{
			try
			{
				BeginMonitoringWithThrow();
			}
			catch (InvalidOperationException e)
			{
				// TODO narrow this to catching only the exception excplicitly thrown in BeginMonitoringWithThrow
				throw e;
			}
			catch (Exception e)
			{
				ErrorReport.NotifyUserOfProblem(new ShowOncePerSessionBasedOnExactMessagePolicy(),
					e, "There was a problem starting up volume monitoring.");
			}`